spec/acceptance/vhost_spec.rb

```diff
@@ -1197,4 +1197,37 @@ class { 'apache::mod::shib': }
       it { is_expected.to contain 'ShibCompatValidUser On' }
     end
   end
+
+  describe 'auth_oidc', unless: (os[:family] == 'ubuntu' && os[:release].to_f == '14.04') do
```

Comment:
```suggestion
  describe 'auth_oidc', unless: (os[:family] == 'ubuntu' && os[:release].to_f == 14.04) do
```

This was my fault for erroneously including quotes around the 14.04... floats are not strings...

Sorry!

Reply:
I haven't noticed the mistake either, thanks!